javasetter/javasetter.go

```diff
@@ -64,18 +65,37 @@ func (j JavaSetter) SetJava(version JavaVersion) (Result, error) {
 	}
 }
 
+func (j JavaSetter) tryToInstallOneOf(versions []string) error {
```

Comment:
nit: I would add a link to the jenv issue tracker so that in the future we'll remember why this was needed.